pageserver/src/metrics.rs

```diff
@@ -2474,6 +2474,64 @@ pub(crate) mod tenant_throttling {
     }
 }
 
+pub(crate) mod disk_usage_based_eviction {
+    use super::*;
+
+    pub(crate) struct Metrics {
+        pub(crate) tenant_collection_time: Histogram,
+        pub(crate) tenant_layer_count: Histogram,
+        pub(crate) layers_collected: IntCounter,
+        pub(crate) layers_selected: IntCounter,
+        pub(crate) layers_evicted: IntCounter,
+    }
+
+    impl Default for Metrics {
+        fn default() -> Self {
+            let tenant_collection_time = register_histogram!(
+                "pageserver_dube_tenant_collection_seconds",
+                "Time spent collecting layers from a tenant -- not normalized by collected layer amount",
+                vec![0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1.0, 5.0, 10.0]
+            )
+            .unwrap();
```

Comment:
Please don't use a random acronym that you made up (`dube`) which could also be confused for `duplicate` by some readers.

If you insist on an acronym, make it all-caps, i.e., `pageserver_DUBE_...` . But it already looks so weird & against conventions, I would really prefer it spelled out.

That way, fuzzy autocomplete in grafana will find it, etc.

Also, the help text should mention disk-usage-base eviction, e.g.


```suggestion
            let tenant_collection_time = register_histogram!(
                "pageserver_disk_usage_based_eviction_tenant_collection_seconds",
                "Time that disk-usage-based eviction spent collecting layers from a tenant -- not normalized by collected layer amount",
                vec![0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1.0, 5.0, 10.0]
            )
            .unwrap();
```

Reply:
Same applies to other metrics.

Reply:
Ok, will expand.

Reply:
[0101b55](https://github.com/neondatabase/neon/pull/6131/commits/0101b55ae5738b536fd0769e439e2e3a34f70392)

failed to repeat it in description. I don't think anyone reads the descriptions without knowing what metric they are describing.